### PR TITLE
DOCS-2382: Update readme

### PR DESCRIPTION
**Pull request by JessamyT**

### CODE REVIEW

**Review by purplenicole730 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -22,6 +26,12 @@ Make sure your project meets the minimum requirements:
 
 `flutter pub add viam_sdk`
 
+## Upgrading
+
+`flutter pub upgrade viam_sdk`
+
+Note that breaking changes may occur in patch version changes, so we recommend locking your project to a specific version by adding a specific version number (for example `viam_sdk: 0.0.12`) to your `pubspec.yaml` file.
```
Comment (njooma) [inline README.md:33]: I would mention that breaking changes may occur in patch version changes BECAUSE we're still in beta.

**Review by njooma — APPROVED** [review]
Small nit but otherwise looks good

File: README.md (hunk 1 above)
Comment (JessamyT) [reply README.md:33]:
```suggestion
Note that the TypeScript SDK is still in beta, so breaking changes may occur in patch version changes.
We recommend locking your project to a specific version by adding a specific version number (for example `viam_sdk: 0.0.12`) to your `pubspec.yaml` file.
```

File: README.md [hunk 2]
```diff
@@ -22,6 +26,13 @@ Make sure your project meets the minimum requirements:
 
 `flutter pub add viam_sdk`
 
+## Upgrading
+
+`flutter pub upgrade viam_sdk`
+
+Note that breaking changes may occur in patch version changes because the TypeScript SDK is still in beta.
```
Comment (andf-viam) [inline README.md:33]:
```suggestion
Note that breaking changes may occur in patch version changes because the Viam Flutter SDK is still in beta.
```

**Review by andf-viam — APPROVED** [review]
LGTM mod one fix. Thank you!

File: README.md (hunk 2 above)
Comment (JessamyT) [inline README.md:33]:
```suggestion
Note that breaking changes may occur in patch version changes because the Flutter SDK is still in beta.
```

**Pull request merged**